docs/data-sources/compliance_framework.md

```diff
@@ -0,0 +1,75 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "mondoo_compliance_framework Data Source - terraform-provider-mondoo"
+subcategory: ""
+description: |-
+  Data source to return compliance frameworks in a Space
+---
+
+# mondoo_compliance_framework (Data Source)
+
+Data source to return compliance frameworks in a Space
+
+## Example Usage
+
+```terraform
+data "mondoo_compliance_framework" "frameworks" {
+  space_id = "your-space-1234567"
+  # space_mrn    = "your-space-mrn" 
+}
+
+output "framework_mrn" {
+  value       = [for framework in data.mondoo_compliance_framework.frameworks.compliance_frameworks : framework.mrn]
+  description = "The MRN of the frameworks in the space."
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `space_id` (String) Space ID
```

Comment:
Not sure if we can highlight that one of them is actually required

Reply:
@chris-rock we at least didn't do so in any other data source where we would have the same situation

Reply:
Only because we have not done it before does not mean it doesn't make sense to do so :-) It is not a blocker but would be nice to have as user.

Reply:
@chris-rock I could not find a solution right now. I suggest, we address this in a separate PR.

Reply:
@chris-rock I could not find a solution right now. I suggest, we address this in a separate PR.

Reply:
Thank you for checking. I agree we can do this as a follow up improvement.